Observer/AfterPlaceOrderObserver.php

```diff
@@ -79,6 +82,13 @@ private function updateOrderLinkStatus(Payment $payment)
         $lastQuoteId = $order->getQuoteId();
         $rzpPaymentId  = $payment->getLastTransId();
 
+        $amount_paid = number_format($this->rzpMethod->getAmountPaid($rzpPaymentId) / 100, 2, ".", "");
+
+        $order->addStatusHistoryComment(
+                    __('Actual Amount Paid of %1.',  $order->getBaseCurrency()->formatTxt($amount_paid))
```

Comment:
@ramth05 There is a mismatch in the amount captured and order notes amount. Please check.

Reply:
@ChetanGN , Yes. We just want to show additional comments of actual amount paid. Capture amount is kind of order Amount.